js/utils.js

```diff
@@ -363,6 +363,16 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     }
   }
 
+  function accessibilityDetails(event, item) {
+    if (event.key === 'Enter' || event.type === 'click') {
+      if (event.type === 'click') {
+        item.removeClass('focus-outline');
```

Comment:
Why does the `focus-outline` class need to be removed?

Reply:
because when the user clicks on the element with the mouse, this class is also activated as I understand it, we only need to add the class when the keyboard event is pressed

Reply:
I believe we rely on the `focus-outline` class to be available so that when user focusses on the UI using **TAB**, the UI will be correctly outlined according to the theme settings.

This means that if `focus-outline` is removed after a user clicks on the UI, then subsequent focuses using **TAB** will not be correctly outlined.

Reply:
Yes, your right. I can remove this code but then we need to make changes in the logic of the `focus-outline` class, because this class has pseudo class focus-within and it shows border when event click

Reply:
@tonytlwu Should I remove this part of the code?

Reply:
@AndrRyaz I'm reviewing the current solution and class names used to manage focus outlines. I'll let you know the results when I'm done cc @sofiiakvasnevska